includes/class-utils.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Automattic\LegacyRedirector;
+
+final class Utils {
+
+    /**
```

Comment:
This line has 4 space indentation, rather than 1 tab.

Reply:
fixed